bin/pump-register-user

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env node
-
 // -*- mode: javascript -*-
 
-// registeruser.js
+// pump-register-user
+//
+// register a new user
 //
-// Register a new user with the activity pump
+// first call register-app, then use this tool
```

Comment:
This information is helpful, but doesn't really say what the file does. I think the old description is better, the new description is better in the README (where you've already put it).